compiler/damlc/lib/DA/Cli/Options.hs

```diff
@@ -225,7 +227,7 @@ enableScenariosOpt = EnableScenarios <$>
 
 dlintEnabledOpt :: Parser DlintUsage
 dlintEnabledOpt = DlintEnabled
-  <$> strOption
+  <$> Options.Applicative.strOption
```

Comment:
Could you leave a comment why strOptionOnce here is not the way to go?

Reply:
done